lblod/frontend-form-builder
# ⚠️ SFB-179: POC begrotingstabel

## ID
 [SFB-179](https://binnenland.atlassian.net/browse/SFB-179)

 ## Description

We want to be able to add a calculation action on a column of a table. This with the help of a field that will calculate all the values that are connected to  this calculation outcome field.

The usecase for this is the `begrotingstabellen` here we want to create table that will include expenses or other type of number values and show the total of all those values in another field. For example this makes it easy to see the differences between the income and expense table.

⚠️ As a POC the code that is being written cannot be merged with master as this is code that is not optimized. ⚠️

 ## Type of change


 ## How to test

1. add a field with displaytype 'calculationOutcomeField' 
2. create a table with one column that will add rows with a fieldtype of numerical input or currency input
3. Go to the configuration tab and select the 'Tabellen' pill
4. Select your table column  and the 'Som' action
5. Switch to the code tab and see if a scope a path a generator is added
6. Try adding some rows to your table and fill in the values. The total of the values wil be shown in the calculationOutcomeField

 ## Links to other PR's

 - https://github.com/lblod/frontend-form-builder/pull/119
